ci-operator/config/openshift/release/openshift-release-master__nightly-4.19.yaml

```diff
@@ -1170,7 +1170,7 @@ tests:
       - observers-resource-watch
     workflow: openshift-e2e-gcp-etcd-scaling
 - as: e2e-gcp-custom-dns-techpreview
-  interval: 168h
+  interval: 24h
```

Comment:
Hi @sadasu may I know why we need to increase the frequency to once a day? Because in my understanding, it's better to maintain a similar frequency for the jobs defined in this file so that the various test states in CR will not deviate. Otherwise, the test results of jobs with high frequency will seriously affect the overall pass rate.(Although we've moved the custom-dns job to jobTier:candidate )